web/vhost.go

```diff
@@ -215,7 +230,12 @@ func (sc *ServiceConfig) syncVhosts(shutdown <-chan interface{}) error {
 				close(ch)
 			}
 			return nil
-		default:
+		case err := <-watchStopped:
+			if err != nil {
+				glog.Infof("VHostRegisty Watch Restarting due to %v", err)
```

Comment:
None of the logging events happen very often. When things stop working it is hard to track down.